### PR TITLE
#25 database manager method connect

### PR DESCRIPTION
**Pull request by AVM82**
connect to #25

### CODE REVIEW

File: src/main/java/juja/sqlcmd/DatabaseManager.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package juja.sqlcmd;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private static final String PORT = "5432";
+    private static final  String SERVER = "localhost";
+    private Connection connection;
```
Comment (Shtramak) [inline src/main/java/juja/sqlcmd/DatabaseManager.java]: Pay your attention on whitespaces (http://bit.ly/2j2qPE1)

File: src/main/java/juja/sqlcmd/DatabaseManager.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    public boolean connect(String database, String user, String password) {
+        String dbConnectionUrl = "jdbc:postgresql://" + SERVER + ":" + PORT + "/" + database;
+        try {
+            Class.forName("org.postgresql.Driver");
+            connection = DriverManager.getConnection(dbConnectionUrl, user, password);
+            return true;
+        } catch (ClassNotFoundException e) {
+            System.err.println("The driver was not found.");
```
Comment (Shtramak) [inline src/main/java/juja/sqlcmd/DatabaseManager.java]: There's no requirement in task to print anything to console (http://bit.ly/2jRFHEn)

**Review by Shtramak — APPROVED** [review]

File: src/main/java/juja/sqlcmd/DatabaseManager.java (hunk 1 above)
Comment (AVM82) [reply src/main/java/juja/sqlcmd/DatabaseManager.java]: Thanks for your remark.

File: pom.xml [hunk 3]
```diff
@@ -19,6 +19,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.1.4</version>
+            </dependency>
```
Comment (ivshapovalov) [inline pom.xml]: Don't forget alignment and blank lines.

File: src/main/java/juja/sqlcmd/DatabaseManager.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            return false;
```
Comment (ivshapovalov) [inline src/main/java/juja/sqlcmd/DatabaseManager.java]: I think thare is no necessity to write several returns.
Maybe place return statement after try-catch block?

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
+package juja.sqlcmd;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
```
Comment (ivshapovalov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java:8]: Two blank lines for what purpose?

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Test
+    public void testConnect() {
+
+        assertTrue(databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd"));
+    }
+
+    @Test
+    public void testConnectIfDatabaseNotExist() {
```
Comment (ivshapovalov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: 'Exists' maybe right?

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    private DatabaseManager databaseManager;
+
+
+    @Before
+    public void setup() {
+        databaseManager = new DatabaseManager();
+    }
+
+    @Test
+    public void testConnect() {
```
Comment (ivshapovalov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: Please try to rename method

**Review by ivshapovalov — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (AVM82) [reply pom.xml]: I will fix it.

File: src/main/java/juja/sqlcmd/DatabaseManager.java (hunk 2 above)
Comment (AVM82) [reply src/main/java/juja/sqlcmd/DatabaseManager.java]: I changed this method

File: src/main/java/juja/sqlcmd/DatabaseManager.java (hunk 4 above)
Comment (AVM82) [reply src/main/java/juja/sqlcmd/DatabaseManager.java]: I changed this method

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 5 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java:8]: Thanks for the comment, I should be more careful

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 7 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: I changed the name of this method to testConnectionWithCorrectParameters

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 6 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: You're right.

**Review by ivshapovalov — APPROVED** [review]

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 8]
```diff
@@ -0,0 +1,36 @@
+package juja.sqlcmd;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertFalse;
```
Comment (Shtramak) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java:6]: Maybe import static org.junit.Assert.assertFalse?

**Review by Shtramak — APPROVED** [review]

**Review by adaleorion — APPROVED** [review]
looks good 👍

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 8 above)
Comment (adaleorion) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java:6]: @AVM82 
Yes I agree with @Shtramak

**Review by kortov — APPROVED** [review]
LGTM

**Pull request merged**